migrations/migrations/20210512142522-add-column-is-admin-user.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+module.exports = {
+  up: (queryInterface, Sequelize) =>
+    queryInterface.addColumn('users', 'is_admin', { type: Sequelize.BOOLEAN, defaultValue: false }),
```

Comment:
Rather than having an `isAdmin` column it would be better to have a `role` column. It is more scalable that way, in case we need to have several roles